.github/workflows/check.yml

```diff
@@ -127,3 +127,51 @@ jobs:
           file: lcov.info
           fail_ci_if_error: false
           token: ${{ secrets.CODECOV_TOKEN }}
+
+  qns-docker-image: # Docker image for QUIC network simulator i.e. interop runner (https://github.com/quic-interop/quic-interop-runner)
+    needs: check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Extract Rust version
+        id: rust-version
+        run: |
+          RUST_VERSION=$(grep -m1 'rust-version' neqo-common/Cargo.toml | cut -d '"' -f 2)
+          echo "::set-output name=version::$RUST_VERSION"
```

Comment:
That's going to run this with our MSRV, rather than the latest.  I would just run on stable.

Reply:
a98aa1edf4e72f9c7778afa66d3056ae296d1e06 updates the workflow to use `stable`.